header.php

```diff
@@ -27,6 +27,46 @@
 
 <section id="main-container">
 
-  <header id="header">
-    <h1><a href="<?php echo home_url(); ?>"><?php bloginfo('name'); ?></a></h1>
+  <header id="header" class="padding-top-small padding-bottom-small">
+    <h1 class="u-visuallyhidden"><?php bloginfo('name'); ?></h1>
+
+    <div class="container">
+      <div class="grid-row">
+        <div class="grid-item item-s-12 item-m-6">
+          <div id="mobile-header" class="site-title">
+            <a href="<?php echo home_url(); ?>">LANZA</a>
+            <div class="mobile-toggle mobile-only">=</div>
+          </div>
+        </div>
+        <div class="grid-item item-s-12 item-m-6 grid-column no-gutter" id="nav-holder">
```

Comment:
then i took the padding off the nav holder, and kept the padding on its children, except removing right padding from the nav elem so i could center its left